rules/linux/macos_local_account.yml

```diff
@@ -0,0 +1,23 @@
+title: Local System Accounts Discovery
+id: ddf36b67-e872-4507-ab2e-46bda21b842c
+status: experimental
+description: Detects enumeration of local systeam accounts on MacOS
+author: Alejandro Ortuno, oscd.community
+date: 2020/10/08
+references:
+  - https://attack.mitre.org/techniques/T1087/001/
+  - https://github.com/redcanaryco/atomic-red-team/blob/master/atomics/T1087.001/T1087.001.md
+logsource:
+  product: macos
+detection:
+  selection:
+    CommandLine|contains:
+      - 'dscl . list /Users'
+      - 'dscacheutil -q user'
```

Comment:
It make sense to split `processname` and `commandline`. It is cheaper from resources point of view (more efficient). 

Here is how santa log looks like for the `dscl . list /Users` command:

```
"process": {
      "args": [
        "dscl",
        ".",
        "list",
        "/Users"
      ],
      "start": "2020-10-10T04:07:08.961Z",
      "pid": 23138,
      "executable": "/usr/bin/dscl",
      "ppid": 21813
    },
```

Here is how santa log looks like for the `dscacheutil -q user` command:

```
"process": {
      "args": [
        "dscacheutil",
        "-q",
        "user"
      ],
      "start": "2020-10-10T04:07:15.514Z",
      "pid": 23155,
      "executable": "/usr/bin/dscacheutil",
      "ppid": 21813
    },
```